src/openhuman/runtime_python_server/server.py

```
@@ -0,0 +1,100 @@
+    except Exception as exc:
+        _emit({"ready": False, "error": f"{type(exc).__name__}: {exc}"})
+        return 1
+
+    _emit({"ready": True, "protocol": PROTOCOL, "backends": ["spacy"]})
+    for line in sys.stdin:
+        line = line.strip()
+        if not line:
+            continue
+        try:
+            req = json.loads(line)
+        except Exception as exc:
```

Ruff (0.15.18)

[warning] Do not catch blind exception: `Exception`

(BLE001)